frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.tsx

```diff
@@ -127,6 +129,10 @@ export default function DataSource(): JSX.Element {
 		}
 	};
 
+	const goToIntegrationsPage = (): void => {
+		history.push('/integrations');
```

Comment:
use the `ROUTES` object here else it's one more place to update if we decide to update the route later. `src/constants/routes.ts` can be found in this file